snuba/query/parser/conditions.py

```diff
@@ -178,9 +179,12 @@ def unpack_array_condition_builder(
         )
 
     def simple_condition_builder(lhs: Expression, op: str, literal: Any) -> Expression:
```

Comment:
I would like to change this as well but we cannot. The query language does not really support unary operators. Conditions like "IS NULL" are represented with a hack as `["column", "IS NULL", None]` since conditions must have three elements.